frontend/src/pages/projects/dataConnections/const.ts

```diff
@@ -3,11 +3,11 @@ import { AWSDataEntry } from '~/pages/projects/types';
 
 export enum AWS_KEYS {
   NAME = 'Name',
-  ACCESS_KEY_ID = 'AWS_ACCESS_KEY_ID',
-  SECRET_ACCESS_KEY = 'AWS_SECRET_ACCESS_KEY',
-  S3_ENDPOINT = 'AWS_S3_ENDPOINT',
-  DEFAULT_REGION = 'AWS_DEFAULT_REGION',
-  AWS_S3_BUCKET = 'AWS_S3_BUCKET',
+  ACCESS_KEY_ID = 'Access-Key',
+  SECRET_ACCESS_KEY = 'Secret-Key',
+  S3_ENDPOINT = 'Endpoint',
+  DEFAULT_REGION = 'Region',
+  AWS_S3_BUCKET = 'Bucket',
```

Comment:
I am pretty sure these should not change. these keys need to have the AWS prefix still

> The underlying values that get injected into the Notebook and are passed to Model Serving will remain the same (prefixed with AWS). This is something we cannot change at this time. This is purely a visual set of changes.

Reply:
Sorry, I totally misunderstood it. Thanks for pointing it out.